spec/grover/processor_spec.rb

```diff
@@ -95,6 +101,23 @@
         end
       end
 
+      context 'when puppeteer-core package only is installed', :remote_browser do
+        before do
+          FileUtils.move 'node_modules/puppeteer', 'node_modules/puppeteer_temp'
+        end
+        after do
+          FileUtils.move 'node_modules/puppeteer_temp', 'node_modules/puppeteer'
+        end
+
+        let(:options) { { 'browserWsEndpoint' => browser_ws_endpoint } }
+        let(:browser_ws_endpoint) { 'ws://localhost:3000/' }
+        let(:url_or_html) { '<html><body style="background-color: blue"></body></html>' }
+
+        it 'does not raise' do
+          expect { convert }.to_not raise_error
+        end
```

Comment:
Specs look good, but I think we could do better with one that also checks that the result actually was a PDF with the expected content. Something like:

```suggestion
        let(:url_or_html) { '<html><body style="background-color: blue">puppeteer-core works!</body></html>' }

        it { expect { convert }.not_to raise_error }
        it { expect(pdf_text_content).to eq 'puppeteer-core works!' }
```